frontend/src/store/slices/tenants/useTenants.test.tsx

```diff
@@ -47,7 +47,7 @@ describe('useTenant slice hook', () => {
   });
 
   afterAll(() => {
-    jest.resetAllMocks();
+    jest.restoreAllMocks();
```

Comment:
I'm curious why this was needed?

Reply:
have a look here for the differences
https://stackoverflow.com/questions/58151010/difference-between-resetallmocks-resetmodules-resetmoduleregistry-restoreallm